src/server/ua_server_binary.c

```diff
@@ -221,6 +221,14 @@ getServicePointers(UA_UInt32 requestTypeId, const UA_DataType **requestType,
         *responseType = &UA_TYPES[UA_TYPES_SETMONITORINGMODERESPONSE];
         break;
 #endif
+#ifdef UA_ENABLE_HISTORIZING
+        // For History read
```

Comment:
Use C-style comments (/* */) instead of C++ (//)

Reply:
Done